.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
+- repo: https://github.com/4Catalyzer/fourmat
+  rev: master
+  hooks:
+    - id: fourmat
```

Comment:
now I remember, the reason I liked the shell way more than this, it's because this _assumes_ that fourmat is in PATH, while it's not necessarily the case. some people might do `entry: fourmat fix`, others `entry: poetry run fourmat fix`, or `entry: pipenv run fourmat fix`, etc.

Reply:
IIUC, pre-commit takes care of creating an isolated virtualenv for fourmat, so it should always be in PATH.

Reply:
wow, magic, how does it even do it? just looking at the setup.py in the repo? because I don't see any other way otherwise

Reply:
Yeah, looks like it uses `pip install .`: https://github.com/pre-commit/pre-commit/blob/e10b81858a9366bed4e59f1fc438405b2020baf4/pre_commit/languages/python.py#L163-L165

Reply:
I got outsmarted :)